api-reference/account-management-api/get-usage.md

```diff
@@ -1,8 +1,8 @@
-# Get account usage API (beta)
+# Get account usage API
 
-{% swagger baseUrl="https://api.imagekit.io" path="/v1/accounts/usage" method="get" summary="Get account usage information (beta)" %}
+{% swagger baseUrl="https://api.imagekit.io" path="/v1/accounts/usage" method="get" summary="Get account usage information" %}
 {% swagger-description %}
-Get the account usage information between two dates.
+Get the account usage information between two dates. However, it is worth noting that this data is cached internally and refreshes every 6 hours.
 {% endswagger-description %}
```

Comment:
Let's move the cache note to a info box

The content can be similar to how we do in other API
```
You can programmatically get the usage information for your ImageKit.io account using this API.

The data in the response has an internal cache with a TTL of 6 hours
```

<img width="1421" alt="image" src="https://github.com/imagekitio/docs-keep-a-changelog/assets/8640877/ceedd4b4-8d23-4f2d-8d5d-0aa655d07c59">

Reply:
Done!